src/routes/safe/components/Transactions/TxList/TxHistoryCollapsed.tsx

```diff
@@ -1,16 +1,16 @@
 import { MultisigExecutionInfo } from '@gnosis.pm/safe-react-gateway-sdk'
-import { ReactElement } from 'react'
+import { ReactElement, useMemo } from 'react'
 
 import { Transaction } from 'src/logic/safe/store/models/types/gateway.d'
-import { useAssetInfo } from './hooks/useAssetInfo'
+import { getAssetInfo } from 'src/routes/safe/components/Transactions/TxList/utils'
 import { useTransactionStatus } from './hooks/useTransactionStatus'
 import { useTransactionType } from './hooks/useTransactionType'
 import { TxCollapsed } from './TxCollapsed'
 
 export const TxHistoryCollapsed = ({ transaction }: { transaction: Transaction }): ReactElement => {
   const nonce = (transaction.executionInfo as MultisigExecutionInfo)?.nonce
   const type = useTransactionType(transaction)
-  const info = useAssetInfo(transaction.txInfo)
+  const info = useMemo(() => getAssetInfo(transaction.txInfo), [transaction.txInfo])
```

Comment:
Can you not just keep the hook, memozing the return value? You won't need to change all instance of this.